lib/plugins/aws/lib/naming.js

```diff
@@ -182,6 +182,15 @@ module.exports = {
   getApiKeyLogicalIdRegex() {
     return /^ApiGatewayApiKey/;
   },
+  getUsagePlanLogicalId() {
+    return 'ApiGatewayUsagePlan';
+  },
+  getUsagePlanKeyLogicalId(usagePlanKeyNumber) {
```

Comment:
Done ✅ I also added one sentence about usage plan quota and throttle to the [API Gateway documentation](https://github.com/serverless/serverless/pull/3715/files#diff-871929e390d61a5d2a3b505ac9b4bf24).